### PR TITLE
Adding re-design for the form and webpage

### PR DESCRIPTION
**Pull request by codesandtags**
Adding redesign

### CODE REVIEW

File: estilos/algoritmo.css [hunk 1]
```diff
@@ -1,105 +1,75 @@
...
+    padding: 0;
 }
 
 header {
-	margin: 0 auto;	
-	margin-bottom: 40px;
-	text-align: center;
+    margin: 0 auto;
+    margin-bottom: 10px;
+    background-image: url(../img/math-header.jpg);
```
Comment (GomezMimo) [inline estilos/algoritmo.css:7]: Can you sort alphabetically the css class, please?

File: estilos/algoritmo.css [hunk 2]
```diff
@@ -1,105 +1,75 @@
...
+    margin: 0;
+    padding: 0;
 }
 
 header {
-	margin: 0 auto;	
-	margin-bottom: 40px;
-	text-align: center;
+    margin: 0 auto;
+    margin-bottom: 10px;
```
Comment (GomezMimo) [inline estilos/algoritmo.css]: What about if you put all the margin attributes in just on one line?

File: estilos/algoritmo.css [hunk 3]
```diff
@@ -1,105 +1,75 @@
...
+    background-image: url(../img/math-header.jpg);
+    background-size: cover;
+    height: 120px;
 }
 
-section	{
-	margin: 0 auto;	
-	width: 500px;
+header h1 {
```
Comment (GomezMimo) [inline estilos/algoritmo.css:14]: Can you sort alphabetically the css class, please?

File: scripts/proceso.js [hunk 4]
```diff
@@ -13,6 +13,7 @@ function cambiarTipoAlgoritmo() {
 }
 
 function obtenerDatosProceso() {
+
```
Comment (GomezMimo) [inline scripts/proceso.js:16]: Is there a reason for this new line?

File: estilos/algoritmo.css (hunk 2 above)
Comment (codesandtags) [reply estilos/algoritmo.css]: Done! thank you my fish!

File: estilos/algoritmo.css (hunk 1 above)
Comment (codesandtags) [reply estilos/algoritmo.css:7]: Done!

File: estilos/algoritmo.css (hunk 3 above)
Comment (codesandtags) [reply estilos/algoritmo.css:14]: Done!

File: estilos/algoritmo.css (hunk 2 above)
Comment (codesandtags) [reply estilos/algoritmo.css]: Maybe in a future fix

**Review by codesandtags — COMMENTED** [review]
Added some changes in the review

**Pull request merged**